website/content/docs/connect/ca/index.mdx

```diff
@@ -34,9 +34,22 @@ services.
 - the Leaf Cert Client Agent is created by auto-encrypt and auto-config. It is used by
 client agents for HTTP API TLS, and for mTLS for RPC requests to servers.
 
-Any secondary datacenters receive an intermediate certificate, signed by the Primary Root
-CA, which is used as the CA certificate to sign leaf certificates in the secondary
-datacenter.
+Any secondary datacenters use their CA provider to generate an intermediate certificate
+signing request (CSR) to be signed by the Primary Root CA. They receive an intermediate
+CA certificate which is used to sign leaf certificates in the secondary datacenter.
+
+It is possible to use different providers across primary and secondary datacenters.
+For example, an operator may use a Vault CA provider for extra security in the primary
+datacenter but choose to use the built-in CA provider in the secondary datacenter which
+may not have a reachable Vault cluster. The pros and cons of both providers are listed below.
+
+## CA Provider Comparison
+
+|            | Consul built-in                    | Vault                                                                             |
+|------------|------------------------------------|-----------------------------------------------------------------------------------|
+| Security   | CA private keys are stored on disk | CA private keys are stored in Vault and are never exposed to Consul server agents |
+| Resiliency | No dependency on external systems; if Consul is available, it can sign certs | Dependent on Vault availability         |
+| Latency    | Certs are signed locally           | Cert signing operations require a network call to Vault                           |
```

Comment:
```suggestion
| Resiliency | No dependency on external systems. If Consul is available, it can sign certificates | Dependent on Vault availability         |
| Latency    | Consul signs certificates locally           | A network call to Vault is required to sign certificates                           |
```
Per style guide, we should avoid using abbreviations.